annotations/README.md

```diff
@@ -0,0 +1,75 @@
+# Annotations
+
+## Setup
+
+Getting this package to work requires the following additions to a gradle build file:
+```gradle
+plugins {
+    ...
+    id "io.freefair.aspectj.post-compile-weaving" version "6.4.3"
+}
+
+dependencies {
+    ...
+    implementation project(':annotations')
+    implementation "org.aspectj:aspectjweaver:1.9.8.RC3"
+}
+
+```
+
+## Usage
+
+This pacakge adds `@CountMetric` and `@ExecutionTimeMetric` method annotations.
+
+### @CountMetric
+@CountMetric will log a value of 1 whenever it is triggered.
+
+### @ExecutionTimeMetric
+@ExecutionTimeMetric will log the execution time of a method whenever it is triggered.
+
+### Annotation Parameters
+
+- `String Name` (Default: `""`)
+    - The name the metric is published under
+    - If the name is left null it will be replaced with `"<ClassName>.<MethodName>.<AnnotationType>"` when being sent to CWL
```

Comment:
I assume if name is default with just an empty string, it will also be replaced with `"<ClassName>.<MethodName>.<AnnotationType>"` as well?